src/providers/ipa/ipa_s2n_exop.c

```diff
@@ -1490,8 +1493,32 @@ static void ipa_s2n_get_list_next(struct tevent_req *subreq)
     if (ret != EOK) {
         DEBUG(SSSDBG_CRIT_FAILURE,
               "Object [%s] has no SID, please check the "
-              "ipaNTSecurityIdentifier attribute on the server-side",
+              "ipaNTSecurityIdentifier attribute on the server-side\n",
               state->attrs->a.name);
+        /* In IPA IPA trust case, the IPA user private group will not contain a
+         * SID, so ignore processing it and continue */
+        if (req_inp->type == REQ_INP_NAME &&
+            strcasecmp(state->attrs->domain_name, state->dom->name) == 0 &&
+            /* user private group name == username */
+            strncasecmp(req_inp->inp.name, state->attrs->a.name, strlen(req_inp->inp.name)) == 0) {
```

Comment:
To be on the safe side I think it would be good to check if the returned object is really a group because the get_list request might be use to resolve a list of users (e.g. group members) as well. I guess `state->attrs->response_type` should be `RESP_GROUP` for a group here.